text/0000-ansi-erase.md

```diff
@@ -0,0 +1,84 @@
+- Feature Name: Add erase codes to term package
+- Start Date: 2022-10-26
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+Add erase left and erase line codes to the term package.
+
+# Motivation
+
+We currently support "erase to the right" but not "erase to the left" and
+"erase the entire line". Given these are all "erase" functions and we already
+have an `erase` method on the `Ansi` primitive in the term package it was
+decided that we needed an RFC before adding them as there are multiple ways
+to go about adding them.
+
+# Detailed design
+
+Three new public primitives will be created in the `term` package that will be
+used to control which direction `erase` erases aka, what code it will return.
+
+```pony
+primitive EraseLeft
+primitive EraseLine
+primitive EraseRight
+
+type _EraseDirection is (EraseLeft | EraseLine | EraseRight)
```

Comment:
What's the motivation for making this a private type alias?

Reply:
To make reading the erase method a little easier so that there isn't the full union listed there but also not exposing this type outside of the package if someone does `use "term"`. That is all.